cmd/guacgql/cmd/ingest.go

```diff
@@ -49,6 +49,7 @@ func ingestData(port int) {
 	ingestVulnerability(ctx, gqlclient)
 	ingestPkgEqual(ctx, gqlclient)
 	ingestCertifyBad(ctx, gqlclient)
+	ingestCertifyBads(ctx, gqlclient)
```

Comment:
adding a blocking comment here, while I update.

Reply:
Updated to make `pkgMatchType` mandatory and fix https://github.com/guacsec/guac/issues/1007